src/Support/Http/Controllers/GraphQLController.php

```diff
@@ -2,21 +2,24 @@
 
 namespace Nuwave\Lighthouse\Support\Http\Controllers;
 
-use GraphQL\Executor\ExecutionResult;
 use Illuminate\Http\Request;
 use Illuminate\Http\Response;
-use Illuminate\Routing\Controller;
 use Nuwave\Lighthouse\GraphQL;
-use Nuwave\Lighthouse\Schema\Context;
-use Nuwave\Lighthouse\Schema\Extensions\ExtensionRegistry;
-use Nuwave\Lighthouse\Schema\Extensions\ExtensionRequest;
+use Illuminate\Routing\Controller;
+use GraphQL\Executor\ExecutionResult;
 use Nuwave\Lighthouse\Schema\MiddlewareRegistry;
+use Nuwave\Lighthouse\Schema\Extensions\ExtensionRequest;
+use Nuwave\Lighthouse\Schema\Extensions\ExtensionRegistry;
+use Nuwave\Lighthouse\Support\Contracts\CreatesContext as Context;
```

Comment:
This alias combined with the property name `$context` misleads into thinking this already is the fully constructed context. How about we omit the aliasing and just call the variable `$createsContext`?